facia-press/app/frontpress/PressedCollectionDeduplication.scala

```diff
@@ -43,7 +43,14 @@ object PressedCollectionDeduplication {
 
   def makeNewBackfill(collectionV: PressedCollectionVisibility, preceedingCollectionVsDeduplicated: Seq[PressedCollectionVisibility]): List[PressedContent] = {
     // We want to remove from the current collection' backfilled's PressedCollections those with a header that has already been used
-    val lookUpCuratedDepth = 3
+
+    // 13th December
+    // This refactoring is meant to handle the following case:
+    // If the current collection has no curated elements, then we make sure that there is no duplicate of any previous curated element.
+    // This is actually meant to always be the case, but we are not currently fully doing it due to the Most Popular container.
+    // This is temporary before a later refactoring.
+
+    val lookUpCuratedDepth = if (collectionV.pressedCollection.curated.isEmpty) 99 else 3
```

Comment:
What does the 99 number represent here? It feels like a bit of a magic number! I wonder if extracting it out to a constant and giving it a name would help? Or could we use a different type for `lookUpCuratedDepth` (like an `Option[Int]` for example)?

Reply:
Yeah, sorry about that. It was a bit magic.